File: main/src/main/java/net/fwitz/math/main/binary/escape/halley/HalleysMethodJedSoftExample1.java
package net.fwitz.math.main.binary.escape.halley;

import net.fwitz.math.main.binary.complex.functions.examples.simple.JedSoftExample1;

public class HalleysMethodJedSoftExample1 {
    public static void main(String[] args) {
        HalleysMethodPlot.render(JedSoftExample1.FN_NAME + " (Halley's Method)", JedSoftExample1::fn);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/halley/HalleysMethodJedSoftExample1.kt
package net.fwitz.math.main.binary.escape.halley

import net.fwitz.math.main.binary.complex.functions.examples.simple.JedSoftExample1

object HalleysMethodJedSoftExample1 {
    @JvmStatic
    fun main(args: Array<String>) = HalleysMethodPlot.render(
        name = JedSoftExample1.FN_NAME + " (Halley's Method)",
        f = JedSoftExample1::fn
    )
}